core/src/test/java/org/infinispan/container/versioning/AbstractClusteredWriteSkewTest.java

```diff
@@ -301,19 +301,22 @@ private void doTestWriteSkewWithPassivation(boolean executeOnPrimaryOwner, Opera
       }
 
       DataContainer dataContainer = TestingUtil.extractComponent(primaryOwner, DataContainer.class);
-      MagicKey[] keys = new MagicKey[MAX_ENTRIES];
+      int retryCount = 5;
       int insertCount = 10;
-      for (int i = 0; i < insertCount; ++i) {
-         for (int j = 0; j < MAX_ENTRIES; ++j) {
-            MagicKey tempKey = keys[j];
-            if (tempKey == null) {
-               tempKey = new MagicKey("other-key-" + j, primaryOwner);
-               keys[j] = tempKey;
+      for (int retry = 0; retry < 5; ++retry) {
```

Comment:
Ok, my question was related to the variable name and the commit comment... I really couldn't tell from the code what's failing so as to be retried :)

Could we not use something like `cache.evict(key)` to force the key to be passivated? IMO only tests that exercise the data container directly should care about how many inserts it takes for the data container to passivate an entry.